CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## unreleased
 
+* SharedPreferences
+  * Encrypt shared preferences in BraintreeSharedPreferences (fixes #440)
```

Comment:
nit: We could say "Encrypt all shared preferences data stored by SDK" or something to keep the implementation details internal, since `BraintreeSharedPreferences` is `package-private`.